cosmosdb/cassandra/ps-cassandra-ru-get.ps1

```diff
@@ -0,0 +1,32 @@
+# Get RU for an Azure Cosmos Cassandra API keyspace or table
+$apiVersion = "2015-04-08"
+$resourceGroupName = "myResourceGroup"
+$accountName = "mycosmosaccount"
+$keyspaceName = "keyspace1"
+$tableName = "table1"
+$keyspaceThroughputResourceType = "Microsoft.DocumentDb/databaseAccounts/apis/keyspaces/settings"
+$tableThroughputResourceType = "Microsoft.DocumentDb/databaseAccounts/apis/keyspaces/tables/settings"
+$keyspaceThroughputResourceName = $accountName + "/cassandra/" + $keyspaceName + "/throughput"
+$tableThroughputResourceName = $accountName + "/cassandra/" + $keyspaceName + "/" + $tableName + "/throughput"
+
+# Get the throughput for a keyspace (returns RU/s or error if not set)
+Get-AzResource -ResourceType $keyspaceThroughputResourceType `
+    -ApiVersion $apiVersion -ResourceGroupName $resourceGroupName `
+    -Name $keyspaceThroughputResourceName | Select-Object Properties
+
+if($error[0].Exception.Message.Split(",")[0].Split(":")[1].Replace("`"","") -eq "NotFound")
```

Comment:
The comment calls out that there's an error, so this can be removed. Going to do a quick edit on the PR to remove the exception catch.